exercises/concept/chessboard/.meta/config.json

```diff
@@ -9,6 +9,7 @@
       "exercism_username": "tehsphinx"
     }
   ],
+  "uuid": "b5f1c789-adf6-487c-95b0-c6bce23d711b",
```

Comment:
The UUID should only be specifiec in the track's `config.json` file. See https://github.com/exercism/docs/blob/main/anatomy/tracks/concept-exercises.md#file-metaconfigjson for the spec of this file.

```suggestion
```